homeassistant/components/media_player/__init__.py

```diff
@@ -808,6 +816,14 @@ def __init__(self, component):
         if not authenticated:
             return web.Response(status=401)
 
+        if player.media_image_remotely_accessible:
+            url = player.media_image_url
+            if url is not None:
+                return web.Response(status=301, headers={
```

Comment:
Do the parameters used in `/api/media_player_proxy` (`token` or `cache`) change whenever the media changes?  If not, we should be using a `302` instead so browsers don't cache the redirect and pin to the image.

Reply:
You're so right.